admin/sql/CreateFunctions.sql

```diff
@@ -1079,4 +1079,16 @@ CREATE OR REPLACE FUNCTION track_count_matches_cdtoc(medium, int) RETURNS boolea
 $$ LANGUAGE SQL IMMUTABLE;
 
 COMMIT;
+
+-----------------------------------------------------------------------
+-- edit_note triggers
+-----------------------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION a_insert_edit_note() RETURNS trigger AS $$
+BEGIN
+    INSERT INTO edit_note_recipient (editor_id, edit_note_id) VALUES (NEW.editor, NEW.id);
```

Comment:
`NEW.editor` is the person who left the note, so that actually needs to get the edit author from the `edit` table (join on `NEW.edit = edit.id`).